hydrus/hydraspec/openapi-hydra_parser.py

```diff
@@ -0,0 +1,113 @@
+"""
+Module to take in Open Api Specification and convert it to HYDRA Api Doc
+
+"""
+import yaml
+from doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
+import json
+
+with open("openapi_sample.yaml", 'r') as stream:
+    try:
+        doc = yaml.load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+info = doc["info"]
+desc = info["description"]
+title = info["title"]
+baseURL = doc["host"]
+name = doc["basePath"]
+api_doc = HydraDoc(name, title, desc, name, baseURL)
+definitions = doc["definitions"]
+paths = doc["paths"]
+definitionSet = set()
+classAndClassDefinition = dict()
+hydra_doc = ""
```

Comment:
Please avoid using so many Global variables. You're not using function arguments **at all**. 
It's not a good practice.

Reply:
okay , thank you , i agree , will do .